### PR TITLE
Initial Setup for sentry

### PR DESCRIPTION
**Pull request by YISUSVII**
Initial Setup for sentry @mlopezdetrinidad this configuration will help us to monitor application issues and crashes

### CODE REVIEW

**Comment by 92Sam** [comment]
@YISUSVII hi my friend please, can send a comment into the history for made the review. and thanks!

**Comment by YISUSVII** [comment]
@92Sam it seems that main branch is not working. The error that you can see on circleCI is similar that local error that i have
I will appreciate your assitence on it

File: sentry.client.config.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import * as Sentry from "@sentry/nextjs";
+
+const { name, version } = require('./package.json');
+const SENTRY_APP_RELEASE = `${name}@${version}`;
+
+Sentry.init({
+  dsn: "https://46bd757e8d9c4e128f525015dffd8e97@o910595.ingest.sentry.io/5900878",
```
Comment (92Sam) [inline sentry.client.config.js:7]: @YISUSVII my friend value incoming from?? we can change this on variable into the .env?

thanks :)

File: sentry.server.config.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import * as Sentry from "@sentry/nextjs";
+
+const { name, version } = require('./package.json');
+const SENTRY_APP_RELEASE = `${name}@${version}`;
+
+Sentry.init({
+  dsn: "https://46bd757e8d9c4e128f525015dffd8e97@o910595.ingest.sentry.io/5900878",
```
Comment (92Sam) [inline sentry.server.config.js:7]: @YISUSVII the same of the another comment bro.

thanks

**Review by 92Sam — APPROVED** [review]
LGTM only left another commets, and hot fix.

**Comment by 92Sam** [comment]
PR Closed, new review and addapted to develop branch.

https://github.com/PAIDNetwork/paid-dapp-nextjs/pull/99

**Pull request closed without merging**